2200045/swea.py

```diff
@@ -0,0 +1,123 @@
+# 5431
+T = int(input())
+
+for t in range(1, T+1):
+
+    tot, good = list(map(int, input().split()))
+    good_students = list(map(int, input().split()))
+    bad_students = []
+
+    for a in range(1, tot+1):
+        if a not in good_students:
+            bad_students.append(a)
+
+    print(f'#{t}', *bad_students)
+
+# 2001 
+T = int(input())
+
+for t in range(1, T+1):
+    n, m = map(int, input().split())
+    nxn = [list(map(int, input().split())) for num in range(n)]
+    max_flies = []
+
+    for another in range(n -m +1):
+        for num in range(n -m + 1):
+            for i in range(m):
+                for x in range(m):
+                    max_flies.append(nxn[i + num][x + another])
+
+    kills = 0
+    kills_record = []
+
+    for count in range(1, len(max_flies)+1):
+        if count % (m*m) == 0:
+            kills += max_flies[count -1]
+            kills_record.append(kills)
+            kills = 0
+        else:
+            kills += max_flies[count -1]
+
+    kills_record = sorted(kills_record)
```

Comment:
```suggestion
    kills_record.sort()
```
`sorted` 대신 `sort`를 활용하셔도 좋을 것 같습니다! 😄